standard/expressions.md

```diff
@@ -3360,14 +3433,16 @@ An *anonymous_method_expression* is one of two ways of defining an anonymous fun
 
 ### 12.9.1 General
 
-The `+`, `-`, `!`, `~`, `++`, `--`, cast, and `await` operators are called the unary operators.
+The `+`, `-`, `!` (logical negation [§12.9.4](expressions.md#1294-logical-negation-operator) only), `~`, `++`, `--`, cast, and `await` operators are called the unary operators.
+
+> *Note*: The postfix null-forgiving operator ([§12.8.9](expressions.md#1289-null-forgiving-expressions)), `!`, due to its compile-time and non-overloadable only nature, is excluded from the above list. *end note*
```

Comment:
But it's still a unary operator, right? If someone asked "what are the unary operators in C#" it should still be included. We may not want it in `unary_expression`, but I'm nervous about excluding it completely.

Let's talk about this one more.

Reply:
Topic moved to #1215